bindings/wasm/examples/worker.ts

```diff
@@ -39,6 +39,87 @@ glMatrix.glMatrix.setMatrixArrayType(Array);
 const io = setupIO(new WebIO());
 io.registerExtensions(KHRONOS_EXTENSIONS);
 
+// manifold static methods (that return a new manifold)
+const manifoldStaticFunctions = [
+  'cube', 'cylinder', 'sphere', 'tetrahedron', 'extrude', 'revolve', 'compose',
+  'union', 'difference', 'intersection', 'levelSet', 'smooth', 'ofMesh'
+];
+// manifold member functions (that return a new manifold)
+const manifoldMemberFunctions = [
+  'add', 'subtract', 'intersect', 'decompose', 'warp', 'transform', 'translate',
+  'rotate', 'scale', 'mirror', 'refine', 'setProperties', 'asOriginal',
+  'trimByPlane', 'split', 'splitByPlane'
+];
+// CrossSection static methods (that return a new cross-section)
+const crossSectionStaticFunctions = [
+  'square', 'circle', 'union', 'difference', 'intersection', 'compose',
+  'ofPolygons'
+];
+// CrossSection member functions (that return a new cross-section)
+const crossSectionMemberFunctions = [
+  'add', 'subtract', 'intersect', 'rectClip', 'decompose', 'transform',
+  'translate', 'rotate', 'scale', 'mirror', 'simplify', 'offset'
+];
+// top level functions that construct a new manifold/mesh
+const toplevelConstructors = ['show', 'only', 'setMaterial'];
+const toplevel = [
+  'setMinCircularAngle', 'setMinCircularEdgeLength', 'setCircularSegments',
+  'getCircularSegments', 'Mesh', 'GLTFNode', 'Manifold', 'CrossSection'
+];
+const exposedFunctions = toplevelConstructors.concat(toplevel);
+
+// Setup memory management, such that users don't have to care about
+// calling `delete` manually.
+// Note that this only fixes memory leak across different runs: the memory
+// will only be freed when the compilation finishes.
+
+const memoryRegistry = new Array<Manifold|CrossSection>();
+
+function addMembers(
+    className: string, methodNames: Array<string>, areStatic: boolean) {
+  //@ts-ignore
+  const cls = module[className];
+  const obj = areStatic ? cls : cls.prototype;
+  for (const name of methodNames) {
+    if (name != 'cylinder') {
```

Comment:
I must have missed this in code review - is this intentional? Just want to make sure I can safely remove it now.